.gitignore

```diff
@@ -1,3 +1,4 @@
+_book
```

Comment:
I think this is unneeded because we generate in dist/docs.
If you have a local `_book` it shouldn't be ignored, but rather deleted :)